images/router/haproxy/conf/haproxy-config.template

```diff
@@ -32,6 +35,10 @@ global
 defaults
   # maxconn 4096
   # Add x-forwarded-for header.
+{{ if ne (env "ROUTER_SYSLOG_ADDRESS" "") ""}}
+  option httplog
```

Comment:
Should / Could we provide a way to change or set the Logging format with this? What if the httplog format is not what we want to record, or what if it does not show us enough information to troubleshoot an issue?